api_v3/lib/types/KalturaUserEntry.php

```diff
@@ -159,4 +159,68 @@ protected function doFromObject($srcObj, KalturaDetachedResponseProfile $respons
 		parent::doFromObject($srcObj, $responseProfile);
 	}
 
+	public function validateForInsert($propertiesToSkip = array())
+	{
+		parent::validateForInsert($propertiesToSkip);
+		$this->validatePropertyNotNull("entryId");
+		$this->validateEntryId();
```

Comment:
@yossipapi I don't see why you need all this. There is an "@insertonly" on both the entryId and userId properties of the KalturaUserEntry object, which should make both these properties non-updatable.